### PR TITLE
feat: add markdown editor

### PR DESCRIPTION
**Pull request by frdomovic**
# feat: add markdown editor

### CODE REVIEW

File: src/lib/components/MarkdownEditor.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+};
+
+export const MarkdownEditor = (props) => {
+  const ref = React.useRef(null);
+  useEffect(() => {
+    if (props.selectedEmoji) {
+      const newMd = `${ref.current.getMarkdown()} ${props.selectedEmoji}`;
+      ref.current.setMarkdown(newMd);
+    }
+    if (!props.message) {
```
Comment (MatejVukosav) [inline src/lib/components/MarkdownEditor.js:42]: else if? do you need them both to execute?

File: src/lib/components/MarkdownToolbar.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="16"
+    height="16"
+    className="bi bi-list-ol svg-style1"
+    viewBox="0 0 16 16"
+    onClick={onClick}
+  >
+    <path
+      fill-rule="evenodd svg-style1"
```
Comment (MatejVukosav) [inline src/lib/components/MarkdownToolbar.js:53]: use react properties

File: src/lib/components/MarkdownToolbar.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="16"
+    height="16"
+    className="bi bi-list-ul svg-style1"
+    viewBox="0 0 16 16"
+    onClick={onClick}
+  >
+    <path
+      fill-rule="evenodd"
```
Comment (MatejVukosav) [inline src/lib/components/MarkdownToolbar.js:70]: fillRule

File: src/lib/components/MarkdownEditor.js (hunk 1 above)
Comment (frdomovic) [reply src/lib/components/MarkdownEditor.js]: updated , no need for both execution

File: src/lib/vm/vm.js [hunk 4]
```diff
@@ -181,6 +182,7 @@ const ApprovedTagsCustom = {
   GlobalStateProvider: true,
   IpfsImageUpload: false,
   Markdown: false,
+  MarkdownEditor: true,
```
Comment (xilosada) [inline src/lib/vm/vm.js:185]:
```suggestion
  MarkdownEditor: false,
```

**Review by xilosada — APPROVED** [review]

**Pull request merged**